float64 position_pan             # set pan position
float64 position_tilt            # set tilt position
---